api/package.json

```diff
@@ -16,7 +16,7 @@
     "unzip-stream": "0.3.0"
   },
   "devDependencies": {
-    "@types/node": "13.1.4",
+    "@types/node": "17.0.44",
```

Comment:
Technically this is using 16.x at runtime

Reply:
Oh should I revert to 13.1.4 or change the version to something in 16.x?